js/ui/codeView.js

```diff
@@ -976,6 +980,8 @@ var CodingSession = new Lang.Class({
         // Only want to minimize if we weren't already minimized.
         if (toMini && !toMini.meta_window.minimized)
             toMini.meta_window.minimize();
+
+        this.emit('minimized', actor);
```

Comment:
This and the other signal pass the actor but the signals are defined without parameters and you don’t use the actor in the callback, so should be removed

Reply:
Removed, that's there because I changed the signal and didn't update that code.